mandatory/1-fix-functions.js

```diff
@@ -32,7 +32,7 @@ function greaterThan10(num) {
 
 function get3rdIndex(arr) {
   let index = 3;
-  let element;
+  let element=arr[index];
 
   return element;
 }
```

Comment:
Correct.

You could also just do `return arr[index]` but your way is also acceptable.